vnext/ReactUWP/Views/ActivityIndicatorViewManager.cpp

```diff
@@ -36,7 +37,11 @@ XamlView ActivityIndicatorViewManager::CreateViewCore(int64_t tag) {
 void ActivityIndicatorViewManager::UpdateProperties(
     ShadowNodeBase *nodeToUpdate,
     const folly::dynamic &reactDiffMap) {
-  auto progressRing = nodeToUpdate->GetView().as<winrt::ProgressRing>();
+
+  const auto progressRing =
+      XamlDirectInstance::GetXamlDirect().GetXamlDirectObject(
```

Comment:
XamlDirectInstance::GetXamlDirect() everywhere seems a bit verbose and redundant in name, XamlDirectInstance::Get() or just have a global func GetXamlDirect() in the react uwp namespace

Reply:
I think these new changes address the verbosity